### PR TITLE
[fix](multi-catalog) add max compute custom odps and tunnel url

### PR DESCRIPTION
**Pull request by wsjz**
## Proposed changes

add max compute custom odps and tunnel url

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1974,6 +1974,18 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_query_hive_views = true;
 
+    /**
+     * ODPS server url of max compute, set it to replace default url.
+     */
+    @ConfField(mutable = true)
+    public static String max_compute_odps_url = "";
```
Comment (morningman) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1981]: No need, remove these 2 config.

**Review by morningman — COMMENTED** [review]
Please update the document

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
run buildall

**Comment by wsjz** [comment]
run buildall

File: be/src/runtime/descriptors.h [hunk 2]
```diff
@@ -238,6 +238,8 @@ class MaxComputeTableDescriptor : public TableDescriptor {
     const std::string region() const { return _region; }
     const std::string project() const { return _project; }
     const std::string table() const { return _table; }
+    const std::string odps_url() const { return _odps_url; }
```
[the remark this answers is not in the document]
Comment (wsjz) [reply be/src/runtime/descriptors.h:241]: ignore

File: be/src/runtime/descriptors.h [hunk 3]
```diff
@@ -238,6 +238,8 @@
...
+    const std::string tunnel_url() const { return _tunnel_url; }
```
[the remark this answers is not in the document]
Comment (wsjz) [reply be/src/runtime/descriptors.h:242]: ignore

**Review by wuwenchi — APPROVED** [review]

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**